From c0686b57e6726e4ef4c05c19fed7bc0880298e21 Mon Sep 17 00:00:00 2001
From: Brian Kim <brian.kim@hardkernel.com>
Date: Tue, 24 May 2016 12:59:32 +0900
Subject: [PATCH] ODROID-C2: aocec: Add CEC_OC_DEVICE_VENDOR_ID command

Added CEC_OC_DEVICE_VENDOR_ID command in cec_rx_process() fucntion to
avoid frequent error message print.

Change-Id: Iece6c2a69606c5d9a8eafce2e2de082c4d1ed4ca
Signed-off-by: Brian Kim <brian.kim@hardkernel.com>
---
 drivers/amlogic/cec/hdmi_ao_cec.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/amlogic/cec/hdmi_ao_cec.c b/drivers/amlogic/cec/hdmi_ao_cec.c
index 01b3aaf..7da8bf5 100644
--- a/drivers/amlogic/cec/hdmi_ao_cec.c
+++ b/drivers/amlogic/cec/hdmi_ao_cec.c
@@ -1220,6 +1220,8 @@ static void cec_rx_process(void)
             cec_send_simplink_ack();
         }
         break;
+	case CEC_OC_DEVICE_VENDOR_ID:
+		break;
 
     default:
         CEC_ERR("unsupported command:%x\n", opcode);
